blog(embertimes77): component patterns (#3726)
emberjs/website: source/blog/2018-12-14-the-ember-times-issue-77.md

---
title: The Ember Times - Issue No. XX
author: Chris Ng, the crowd
tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
alias : "blog/2018/12/14-the-ember-times-issue-77.html"
responsive: true
---

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹

> <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>, want to learn more about component patterns 📚🧐? 

Reviewer: qq here just for the purpose of clarity so I don't misunderstand this, I'm assuming `<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>` ideally should be deleted and replaced with the catchy intro

---

## [Components patterns in Ember Blog 📚🧐](https://twitter.com/TheMacsour/status/1072088269742112768)

> [@Exelord](https://github.com/Exelord) published a blog post on [component patterns in Ember](https://medium.com/macsour/components-patterns-in-ember-js-5e6fc6eea28f). In the article, [@Exelord](https://github.com/Exelord) gathered **component patterns** around the Ember community for readers to better understand what kind of component pattern would best match your particular use case.

Reviewer 2: I think since `Exelord` is mentioned in each sentence, this might also be written as follows for more variety:

```suggestion
[@Exelord](https://github.com/Exelord) published a blog post on [component patterns in Ember](https://medium.com/macsour/components-patterns-in-ember-js-5e6fc6eea28f). The article presents a collection of **component patterns** used in the Ember community for readers to better understand what kind of component pattern would best match their particular use case.
```

Also updated `your particular use case` -> `their particular use case`
